stable_baselines3/common/monitor.py

```diff
@@ -1,11 +1,14 @@
-__all__ = ["Monitor", "get_monitor_files", "load_results"]
+# flake8: noqa F401
```

Comment:
why for the whole file? (unused import?)

Reply:
I am not sure... I noticed `get_monitor_files` and others are in `__all__` so I just included everything similarly like

```__all__ = ["Monitor", "ResultsWriter", "get_monitor_files", "load_results"]```